pkg/splunk/client/enterprise.go

```diff
@@ -424,23 +424,23 @@ func (c *SplunkClient) GetClusterMasterInfo() (*ClusterMasterInfo, error) {
 }
 
 // IndexerClusterPeerInfo represents the status of a indexer cluster peer.
-// See https://docs.splunk.com/Documentation/Splunk/latest/RESTREF/RESTcluster#cluster.2Fslave.2Finfo
+// See https://docs.splunk.com/Documentation/Splunk/latest/RESTREF/RESTcluster#cluster.2Fpeer.2Finfo
```

Comment:
Just a comment to make sure these url/anchors still work.

Reply:
Thanks Jeff. Double checked they work as the documentation is already using the word "peer".